src/shared.ts

```diff
@@ -132,6 +132,7 @@ export const initStripe = (
   return stripe;
 };
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
@schau-stripe Can you share more about why this lint ignore was required as the only change to this file? It's not immediately clear to me, and would prefer to avoid ignores if possible.

Reply:
Sure! I'm getting this error with es-lint `Argument 'params' should be typed with a non-any type`

Reply:
Looks like this was a warning-level lint, which existed before this PR. Seems fine to add the ignore comment (or leave it out entirely)

Reply:
Sounds good, thanks for the review!